src/activities/content/ContentEntity.js

```diff
@@ -0,0 +1,13 @@
+import { Entity } from '../../es6/Entity';
+
+/**
+ * ContentEntity class representation of a d2l content entity.
+ */
+export class ContentEntity extends Entity {
+	/**
+	 * @returns {string} Name of the content item
+	 */
+	name() {
```

Comment:
Does this need to become `title` now also?

Reply:
Yes, that would probably be a more logical function name to keep everything consistent. Will adjust accordingly.